src/test/java/com/exasol/adapter/dialects/snowflake/SnowflakeVirtualSchemaIntegrationTestSetup.java

```
@@ -0,0 +1,252 @@
+            final ExasolSchema exasolSchema = this.exasolFactory.createSchema(SCHEMA_EXASOL);
+            this.adapterScript = createAdapterScript(exasolSchema);
+            final String connectionString = getSnowflakeConnectionString(accountName);
+            connectionDefinition = getSnowflakeConnectionDefinition(connectionString, userName, password);
+        } catch (final SQLException | BucketAccessException | TimeoutException exception) {
+            throw new IllegalStateException("Failed to created snowflake test setup.", exception);
+        } catch (final InterruptedException exception) {
+            Thread.currentThread().interrupt();
+            throw new IllegalStateException("Thread was interrupted");
+        } catch (final IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private ConnectionDefinition getSnowflakeConnectionDefinition(final String connectionString, final String username,
+                                                                  final String password) {
+        final ConnectionDefinition connectionDefinition;
+        connectionDefinition = this.exasolFactory.createConnectionDefinition("SNOWFLAKE_CONNECTION", connectionString,
+                username, password);
+        return connectionDefinition;
+    }
+
+    private String getSnowflakeConnectionString(final String accountname) {
+        final String connectionString = "jdbc:snowflake://" + accountname + ".snowflakecomputing.com";
+        return connectionString;
+    }
+
+    private Connection getSnowflakeConnection(final String username, final String password, final String accountname)
+            throws SQLException {
+        try {
+            Class.forName("net.snowflake.client.jdbc.SnowflakeDriver");
+        } catch (final ClassNotFoundException ex) {
+            System.err.println("Driver not found");
+        }
```

Obsolete - class will be loaded automatically.
But if loading explicitly, then just let the exception be raised, don't catch it.

---

Removed try catch and let it bubble up to general snowflake setup failure handling if it fails.